serializations.md

```diff
@@ -15,29 +15,49 @@ user API.
 
 ## Static size object serialization
 
-#### Block header
+There are two types of blocks:
 
-For block headers all fields sizes are statically known and can be
+* key block - does not contain a list of transactions, therefore, it is
+not divided into a header and a list of transactions;
+* micro block - contains a header and a list of transactions.
+
+#### Key block
+
+All fields sizes are statically known and can be constructed directly as
+a byte array.
+
+| Fieldname | Size (bytes) |
+| --- | --- |
+| version    | 8    |
+| height     | 8    |
+| prev_hash  | 32   |
+| state_hash | 32   |
+| miner      | 32   |
+| target     | 8    |
+| pow        | 168  |
+| nonce      | 8    |
+| time       | 8    |
+
+#### Micro block header
+
+For micro block headers all fields sizes are statically known and can be
 constructed directly as a byte array.
 
 | Fieldname | Size (bytes) |
 | --- | --- |
-| version      | 8    |
-| height       | 8    |
-| prev_hash    | 32   |
-| txs_hash     | 32   |
-| root_hash    | 32   |
-| target       | 8    |
-| pow_evidence | 168  |
-| nonce        | 8    |
-| time         | 8    |
-| miner        | 32   |
-
-#### Block
-
-The only difference between a block and its header is the list of
+| version    | 8    |
+| prev_hash  | 32   |
+| state_hash | 32   |
+| txs_hash   | 32   |
+| miner_hash | 32   |
```

Comment:
I understand miner is not strictly needed in microblock as it can be inferred from lastest keyblock linked (potentially transitively) by previous hash: is it kept for easier signature verification?

Reply:
`miner_hash` is not needed, but for now it's included in the micro blocks as it makes it possible to figure out what key block a micro block belongs to. This info could be retrieved from `aec_chain_state`, although, it's not implemented yet.